gateway/src/test/java/org/georchestra/gateway/filter/headers/RemoveHeadersGatewayFilterFactoryTest.java

```diff
@@ -106,10 +106,37 @@ void testIsSecurityHeader() {
         assertTrue(secHeadersConfig.matches("SEC-USER"));
         assertTrue(secHeadersConfig.matches("sec-org"));
         assertTrue(secHeadersConfig.matches("SEC-ORG"));
-        assertTrue(secHeadersConfig.matches("authoriZation"), "Basic Auth header shoulud should be filtered");
+        assertTrue(secHeadersConfig.matches("authoriZation"), "Basic Auth header should should be filtered");
```

Comment:
now we have `should should`

Reply:
you merged before I had the time to fix ;-)